frontend/packages/dev-console/src/components/pipelines/PipelineResources.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {
+  Form,
+  Button,
+  Alert,
+  TextInputTypes,
+  ActionGroup,
+  ButtonVariant,
+} from '@patternfly/react-core';
+import { FormikProps, FormikValues } from 'formik';
+import { ButtonBar } from '@console/internal/components/utils';
+import { MultiColumnField, InputField, DropdownField } from '../formik-fields';
+
+const PipelineResources: React.FC<FormikProps<FormikValues>> = (props) => {
+  enum resourceTypes {
+    '' = 'Select resource type',
+    git = 'Git',
+    image = 'Image',
+    cluster = 'Cluster',
+    storage = 'Storage',
+  }
+  return (
+    <Form onSubmit={props.handleSubmit}>
+      <div className="co-m-pane__form">
+        <MultiColumnField
+          name="resources"
+          addLabel="Add Pipeline Resources"
+          headers={['Name', 'Resource Type']}
+          emptyValues={{ name: '', type: '' }}
+        >
+          <InputField name="name" type={TextInputTypes.text} placeholder="Name" />
+          <DropdownField name="type" items={resourceTypes} fullWidth />
+        </MultiColumnField>
+        <hr />
```

Comment:
This `<hr />` tag is missing in the `PipelineParameters` component. The UXD links added in the Jira stories look different so I am not sure if the horizontal rule has to be there or not. All I am saying is to make it consistent.

Reply:
Checked this on the Environment tab of the Deployment Details Page and it has the horizontal rule

Reply:
Yeah UXD has a horizontal line above the submit reload buttons. I might have missed adding it there too.

Reply:
Done.